### PR TITLE
Possibility to uninstall supervisor

### PR DESCRIPTION
**Pull request by Waldz**
Prepare Supervisor install/uninstall to be moved to Windows installer (as https://github.com/mysteriumnetwork/mysterium-vpn-desktop/pull/36)

### CODE REVIEW

File: supervisor/install/install_windows.go [hunk 1]
```diff
@@ -37,32 +37,35 @@ func Install(options Options) error {
...
 
+	log.Println("Cleaning up previous installation")
+	uninstallService(m, serviceName)
+
 	config := mgr.Config{
 		ServiceType:  windows.SERVICE_WIN32_OWN_PROCESS,
 		StartType:    mgr.StartAutomatic,
 		ErrorControl: mgr.ErrorNormal,
 		DisplayName:  "MystSupervisor Service",
 		Description:  "Mysterium Network dApp supervisor service is responsible for managing network configurations",
```
Comment (tadaskay) [inline supervisor/install/install_windows.go:48]: On windows, this is actually visible to the users, hence I would improve the wording.

```suggestion
		DisplayName:  "MysteriumVPN Supervisor",
		Description:  "Handles network configuration for MysteriumVPN application.",
```

File: supervisor/install/install_windows.go [hunk 2]
```diff
@@ -85,13 +89,14 @@ func uninstallService(m *mgr.Mgr, name string) error {
...
 	s.Control(svc.Stop)
+
 	err = s.Delete()
 	if err != nil {
 		return fmt.Errorf("could not mark service for deletion: %w", err)
 	}
+
 	err = eventlog.Remove(name)
 	if err != nil {
 		return fmt.Errorf("RemoveEventLogSource() failed: %s", err)
```
Comment (zolia) [inline supervisor/install/install_windows.go:102]:
```suggestion
		return fmt.Errorf("remove event log source failed: %w", err)
```

File: supervisor/install/install_windows.go (hunk 1 above)
Comment (Waldz) [reply supervisor/install/install_windows.go]: Improved. 
Dont we want to rename `const serviceName = "MystSupervisor"` also (mac, windows)?

Comment (Waldz) [reply supervisor/install/install_windows.go]: Service name now is `MysteriumVPNSupervisor`

**Pull request merged**